### PR TITLE
Microsoft Graph Security apps doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/microsoft-graph-security.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/send-data/ms-graph.svg')} alt="icon" width="50"/>
+
+The Sumo Logic App for Microsoft Graph Security enables you to identify security threats by analysing alert logs and helps you to improve web security posture, proactively detect and mitigate potential security risks, and improve compliance with security policies and regulations.
+
+Key features of the Microsoft Graph Security app include:
+- Detect and mitigate potential ransomware attacks.
+- Monitor endpoint security and identify potential threats.
+- Have a high-level overview of the organizational security posture, including metrics such as the number of alerts, alert severity, and status.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/microsoft-graph-security.md:16]:
```suggestion
- High-level overview of the organizational security posture, including metrics such as the number of alerts, alert severity, and status.
```

File: docs/integrations/saas-cloud/microsoft-graph-security.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- Allows security teams to identify potential threats and take appropriate action based on the trends and patterns of security alerts over time.
+
+## Log Types
```
Comment (JainM6) [inline docs/integrations/saas-cloud/microsoft-graph-security.md:19]:
```suggestion
## Log types
```

File: docs/integrations/saas-cloud/microsoft-graph-security.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app.
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Microsoft Graph Security Dashboards​​
```
Comment (JainM6) [inline docs/integrations/saas-cloud/microsoft-graph-security.md:137]:
```suggestion
## Viewing Microsoft Graph Security dashboards​​
```

File: docs/integrations/saas-cloud/microsoft-graph-security.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+### Microsoft Graph Security - Alerts Overview
```
Comment (JainM6) [inline docs/integrations/saas-cloud/microsoft-graph-security.md:145]: I think we should keep the dashboard name heading generic, no need to match UI.

**Review by JainM6 — APPROVED** [review]
Approving it, but please check suggestion for the dashboard names heading.

**Comment by JV0812** [comment]
Thanks Megha!
I have kept the dashboard title as generic.

**Pull request merged**